inst/lintr/linters.csv

```diff
@@ -6,6 +6,7 @@ commas_linter,style readability default
 commented_code_linter,style readability best_practices default
 cyclocomp_linter,style readability best_practices default configurable
 equals_na_linter,robustness correctness common_mistakes default
+expect_null_linter,testing best_practices
```

Comment:
We shouldn't introduce tags with only one linter IMO.
What about `package_development` as a more general tag which would also fit, e.g., `package_hooks_linter`?

Reply:
basically agreed, but otoh there will be many more with this tag, see the Google sheet

Reply:
Okay fine then. For this specific tag though I think `package_development` would be more intuitive.
`testing` could also mean `if () {} else {}` or `ifelse()` or `if_else()`.
WDYT?

Anyway we need a new Rd for the new tag, to be placed in `R/linter_tag_docs.R`

Reply:
SGTM, done